chain/client/src/sync/sync_actor.rs

```diff
@@ -4,14 +4,19 @@ use near_network::types::{PeerManagerMessageRequest, StateSyncResponse};
 use near_o11y::{handler_debug_span, OpenTelemetrySpanExt, WithSpanContext};
 use near_performance_metrics_macros::perf;
 use near_primitives::hash::CryptoHash;
-use near_store::ShardUId;
+use near_primitives::state_part::PartId;
+use near_primitives::state_sync::StatePartKey;
+use near_primitives::types::ShardId;
+use near_store::DBCol;
+
+use near_chain::chain::{ApplyStatePartsRequest, ApplyStatePartsResponse};
 use tracing::{debug, info, warn};
 
 /// Message channels
 #[allow(dead_code)]
 struct MessageSenders {
     /// Used to send messages to client
-    client_adapter: Sender<ClientSyncMessage>,
+    client_adapter: Sender<ApplyStatePartsResponse>,
```

Comment:
this change is just for convenience, since just changing it to `ApplyStatePartsResponse` makes this PR work without more refactoring, and if we wanna send it a `ClientSyncMessage` in the future, we can add that back later easily